files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -156,6 +156,10 @@
   cert, key = OmnibusHelper.gen_certificate
 end
 
+# TODO: we are now using public_keys instead of certs and the
+#       bootstrap process now uses the public key. This is still
+#       here for backwards compatibility in case it is used anywhere
+#       else but we should consider deprecating it in the future.
```

Comment:
This search https://github.com/search?utf8=%E2%9C%93&q=%22pivotal.cert%22+user%3Achef&type=Code&ref=searchresults

seems to indicate that the only places using this file are:

- this cookbook
- ominibus-pc which shouldn't be used anywhere.
- ec-metal which seems to be replacing our entire default recipe (cc @irvingpop )
- chef-server-cloudformation-templates which seems to be pulling it from an s3 bucket for some sort of automated installation (cc @juliandunn)

Given that, I don't think we need to reconfigure any apps.  We could likely rip this out completely with minimal impact to our own code.  It could, however, potentially break automation that our customers have built that expect this file to be here.

Unless I missed something in my search above, I'm +1 on merging this and making a task to clean up the cert in the future.